Exception.TASK_NOT_FOUND=La tarea con id {0} no fue encontrada.
Exception.TASK_ALREADY_EXISTS=La tarea con id {0} ya existe.

Message.TASK_SAVED_SUCCESSFULLY=La tarea fue guardada exitosamente.
Message.TASK_UPDATED_SUCCESSFULLY=La tarea fue actualizada exitosamente.
Message.TASK_DELETED_SUCCESSFULLY=La tarea fue eliminada exitosamente.
